world.js

```diff
@@ -25,6 +25,7 @@ const localEuler = new THREE.Euler();
 
 // world
 export const world = {};
+world.reverbZone = [];
```

Comment:
Should be `reverbZones`.

Reply:
[change the reverbZone to reverbZones](https://github.com/webaverse/app/pull/3056/commits/84a5bd2a924343dc99e8f7828d5f14cf85cd4958)